### PR TITLE
Reading time - changes based on feedback

### PR DESCRIPTION
**Pull request by melanierogan**
## Who is this for?

Anyone who reads articles

## What is it doing for them?

Increases space between `|` and reading time text. Uses the phrase 'Average reading time' to step away from being ableist in our terminology, and changes min to minutes.

### CODE REVIEW

File: content/webapp/pages/article.tsx [hunk 1]
```diff
@@ -228,10 +228,22 @@ const ArticlePage: FC<Props> = ({ article, jsonLd }) => {
                   </Space>
                 </Fragment>
               ))}
-            {readingTime && article.readingTime ? (
+            {!readingTime && article.readingTime ? (
```
Comment (rcantin-w) [inline content/webapp/pages/article.tsx:231]: Leftover from testing?

Comment (melanierogan) [reply content/webapp/pages/article.tsx:231]: Gosh! Yes, thank you for that!

File: content/webapp/pages/article.tsx [hunk 2]
```diff
@@ -228,10 +228,22 @@ const ArticlePage: FC<Props> = ({ article, jsonLd }) => {
...
               <>
                 <span className={font('intr', 6)}>
-                  <span className={font('intr', 5)}>| </span> reading time{' '}
+                  <span className={font('intr', 5)}>
+                    {' '}
+                    <Space
```
Comment (rcantin-w) [inline content/webapp/pages/article.tsx:236]: [Alex shared a nice idea for those](https://github.com/wellcomecollection/wellcomecollection.org/pull/8519#discussion_r977915695), maybe it's of interest? 
([the commit with my changes](https://github.com/wellcomecollection/wellcomecollection.org/pull/8519/commits/25d04636351e79673579541f44447a2d60ad21b5))

Comment (rcantin-w) [reply content/webapp/pages/article.tsx:236]: I know yours aren't conditional so you don't need all of it, but it also looks cleaner in the codebase and it doesn't need to be detected by a reader, so an idea!

**Review by rcantin-w — APPROVED** [review]
One comment, but not a blocker :) 

🙌

File: content/webapp/pages/article.tsx [hunk 3]
```diff
@@ -229,12 +237,11 @@ const ArticlePage: FC<Props> = ({ article, jsonLd }) => {
...
               ))}
             {readingTime && article.readingTime ? (
-              <>
-                <span className={font('intr', 6)}>
-                  <span className={font('intr', 5)}>| </span> reading time{' '}
-                  <span className={font('intb', 6)}>{article.readingTime}</span>
-                </span>
-              </>
+              <ReadingTime>
+                {' '}
```
Comment (rcantin-w) [inline content/webapp/pages/article.tsx:241]: np/ I think this could be removed

**Review by rcantin-w — CHANGES_REQUESTED** [review]
Screenshot 2022-09-28 at 12 58 02

There's an extra space that I think is caused by https://github.com/wellcomecollection/wellcomecollection.org/pull/8558/files#diff-2046dc252f32911f880ad3ea39918187f95c8f6cde232a0584f3600c0e4710dcR228, might need a bit of refactoring!

**Comment by melanierogan** [comment]
> Screenshot 2022-09-28 at 12 58 02
> 
> There's an extra space that I think is caused by https://github.com/wellcomecollection/wellcomecollection.org/pull/8558/files#diff-2046dc252f32911f880ad3ea39918187f95c8f6cde232a0584f3600c0e4710dcR228, might need a bit of refactoring!

Do you mean the one after Nicole Collfield? That's a tricky one, it seems to be coming from the `contributor` part. I'm having a look though

**Review by davidpmccormick — APPROVED** [review]

**Review by rcantin-w — APPROVED** [review]
Thank you 🎉 👏

**Pull request merged**